charts/opentelemetry-operator/values.yaml

```diff
@@ -62,6 +62,11 @@ admissionWebhooks:
       # kind:
       # name:
 
+## Create the provided ClusterRoles and ClusterRoleBindings
+##
+rbac:
+  enabled: true
```

Comment:
I think we should use the same interface as we have in the collector chart:

```
clusterRole:
  create: true
```

Reply:
Unless you you think that that interface is not the best (we will need to change collector chart in that case then)

Reply:
@dmitryax changed it to your suggestion